source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection if the value of their
+``pop`` field is less than or equal to *500*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/find.go
+   :start-after: begin find
+   :end-before: end find
+   :emphasize-lines: 4
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
+
+Expected Result
+---------------
+
+This example returns a ``Cursor`` object that contains
+documents similar to these:
```

Comment:
```suggestion
This example creates a ``Cursor`` object that returns
documents similar to these:
```

Issue: I don't believe the above code snippet has a return value, so I think it is incorrect to say that something is returned.

Issue: I do not believe a cursor contains the result documents, but rather acts as an iterable that can retrieve them in batches.

Reply:
accepted